.resinci.yml

```diff
@@ -4,3 +4,6 @@ docker:
     - path: .
       dockerfile: Dockerfile
       docker_repo: balena/open-balena-base
+    - path: .
+      dockerfile: Dockerfile.no-systemd
+      docker_repo: balena/open-balena-base-no-systemd
```

Comment:
@nazrhom I need to publish a different variant of our base image. This way we would need to create a new repo.
Is there a better way? We could also use the same repo but define a tag prefix - is it possible with current balenaCI?

Reply:
@roman-mazur its currently not possible as the tags are used to identify different branches.

Reply:
Ok, falling back to circleCI.